### PR TITLE
EN-13224 bump dc version

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

**Comment by aerust** [comment]
Commit should probably also reference EN-13225 and add bullet point of
3. secondary-watcher updates version on the secondary manifest in a new transaction

File: store-pg/src/test/scala/com/socrata/pg/store/DatabaseTestBase.scala [hunk 1]
```diff
@@ -158,7 +158,7 @@ trait DatabaseTestBase extends Logging {
       if (isNew) secondaryMfst.addDataset(storeId, datasetId)
       secondaryMfst.claimDatasetNeedingReplication(storeId, claimantId, claimTimeout).foreach { job =>
         try {
-          pb(NamedSecondary(storeId, secondary), job)
+          pb(NamedSecondary(storeId, secondary, ""), job)
         } finally {
```
Comment (aerust) [inline store-pg/src/test/scala/com/socrata/pg/store/DatabaseTestBase.scala]: nit: could put "read" instead of "" since it is what we use in prod for pg

Comment (chitang) [inline store-pg/src/test/scala/com/socrata/pg/store/DatabaseTestBase.scala]: really, it isn't pg in prod?

**Comment by aerust** [comment]
other than nit, looks good to me :shell:

**Pull request merged**